stdlib/public/core/Optional.swift

```diff
@@ -487,3 +487,78 @@ extension Optional {
   }
 
 }
+
+//===----------------------------------------------------------------------===//
+// Bridging
+//===----------------------------------------------------------------------===//
+
+#if _runtime(_ObjC)
+extension Optional : _ObjectiveCBridgeable {
+  // The object that represents `none` for an Optional of this type.
+  internal static var _nilSentinel : AnyObject {
```

Comment:
Unfortunately, we don't support stored static properties on generic types. We have to rely on the runtime's caching.